test/plugin/base_test.rb

```diff
@@ -1590,13 +1647,15 @@ def verify_default_log_entry_text(text, i, entry)
   end
 
   # The caller can optionally provide a block which is called for each entry.
-  def verify_json_log_entries(n, params, payload_type = 'textPayload')
-    i = 0
+  def verify_log_entries(n, params, payload_type = 'textPayload')
+    jsonify_log_entries
```

Comment:
I still really dislike having to call this (one, because it's easy to miss, and thus error-prone, and two, because it has no return value, and seems to be modifying the data in-place). The old factory method pattern was a lot clearer, IMO.
In any case, I'd call this `compute_json_log_entries`, or even `compute_logs_sent`.

Reply:
Yeah, that bit could be confusing on a second thought. I've reverted this change.